packages/css-engine/src/core/to-value.ts

```diff
@@ -39,5 +44,26 @@ export const toValue = (
       fallbacks.length > 0 ? `, ${fallbacks.join(", ")}` : "";
     return `var(--${value.value}${fallbacksString})`;
   }
-  return value.value;
+
+  if (value.type === "keyword") {
+    return value.value;
+  }
+
+  if (value.type === "invalid") {
+    return value.value;
+  }
+
+  if (value.type === "unset") {
+    return value.value;
+  }
+
+  if (value.type === "rgb") {
+    return `rgba(${value.r}, ${value.g}, ${value.b}, ${value.alpha})`;
+  }
+
+  // Will give ts error in case of missing type
+  assertUnreachable(value, `Unknown value type`);
+
+  // Will never happen
+  throw new Error("Unknown value type");
```

Comment:
hmm why do you need both, the above should already throw

Reply:
ts don't know that function always throws
<img width="710" alt="image" src="https://user-images.githubusercontent.com/5077042/206740375-38ce797d-b753-43f1-810e-724800e5c70c.png">

There is another exhaustive check variant exists:

```
  // @ts-expect-error check value is never
  value.type;
```
In that case, will be just a final throw, but such a check looks 100x dirtier for uninitiated

Reply:
yes, so keep the one that works with typescript, remove the second one?